cypress/TestCases/FireboltCertification/UserInterest.feature

```diff
@@ -0,0 +1,130 @@
+        And 1st party app invokes the 'Firebolt' API to 'notify requestUserInterest with type <setParam>'
+        Then 'Firebolt' platform responds to '1st party app' with '<method_Content>'
+
+        Examples:
+            | Scenario                         | setParam                     | method_Content                            |
+            | interest and reason playlist     | interest reason playlist     | requestUserInterest with reason playlist  |
+            | interest and reason reaction     | interest reason reaction     | requestUserInterest with reason reaction  |
+            | interest and reason recording    | interest reason recording    | requestUserInterest with reason recording |
+            | disinterest and reason playlist  | disinterest reason playlist  | requestUserInterest with reason playlist  |
+            | disinterest and reason reaction  | disinterest reason reaction  | requestUserInterest with reason reaction  |
+            | disinterest and reason recording | disinterest reason recording | requestUserInterest with reason recording |
+
+    @coreSDK @sdk @transport @userinterest
+    Scenario Outline: Content.requestUserInterest - Negative Scenario: <Scenario> expecting error
+        When 1st party app invokes the 'Firebolt' API to 'get requestUserInterest with <param>'
+        Then 'Firebolt' platform responds to '1st party app' with 'invalid params for content requestUserInterest'
+
+        Examples:
+            | Scenario                               | param                |
+            | Invalid Interest value - test params   | invalid interestType |
+            | Invalid Interest type - boolean params | boolean interestType |
+            | Invalid reason value - test params     | invalid reasonType   |
+            | Invalid Interest type - boolean params | boolean reasonType   |
+    
+    @coreSDK @sdk @userinterest @notSupported
+    Scenario: Content.requestUserInterest - Negative Scenario: Platform-UX - Notify requestUserInterest but platform timeout without sending response
+        When 1st party app invokes the 'Firebolt' API to 'notify requestUserInterest with type interest timeout'
+        Then 'Firebolt' platform responds to '1st party app' with 'not available for requestUserInterest'
+
+    @coreSDK @sdk @userinterest @notSupported
+    Scenario: Content.requestUserInterest - Negative Scenario: Platform-UX - Notify requestUserInterest but 3rd party app return error
+        When 1st party app invokes the 'Firebolt' API to 'notify requestUserInterest with type interest returns error'
+        Then 'Firebolt' platform responds to '1st party app' with 'invalid parameters for requestUserInterest'
+
+    @coreSDK @sdk @userinterest
+    Scenario: Content.requestUserInterest - Negative Scenario: Platform-UX - Notify requestUserInterest without registering for providers
```

Comment:
this TC should also be **@notSupported** right

Reply:
corrected